src/test/scala/org/broadinstitute/dsde/workbench/sam/api/AdminUserRoutesSpec.scala

```diff
@@ -149,6 +149,76 @@ class AdminUserRoutesSpec extends AnyFlatSpec with Matchers with ScalatestRouteT
     }
   }
 
+  "PATCH /admin/v1/user/{userSubjectId}" should "update a user's googleSubjectId" in {
+    // Arrange
+    val samRoutes = new MockSamRoutesBuilder(allUsersGroup)
+      .callAsAdminUser() // enabled "admin" user who is making the http request
+      .withEnabledUser(defaultUser) // "persisted/enabled" user we will check the status of
+      .withAllowedUser(defaultUser)
+      .build
```

Comment:
i am not super happy with how this mock routes builder is setup currently, it feels like we are testing too much of the mock rather than the underlying implementation of the route

Reply:
When it is declared it sets up the state for the specific test so that would mess things up unfortunately